### DISCUSSION TITLE
97. Named Lock에 대해서 설명해주세요.

### DISCUSSION THREAD

**Discussion by jongmee** [jewelry-box/cs-interview#101] [General]
Named Lock에 대해서 설명해주세요.

**Comment by linirini** [comment]
MySQL 엔진 락 중 하나로 잠금의 대상이 레코드, 테이블 같은 데이터베이스의 객체가 아니라 **사용자가 설정한 문자열**이다.
`GET_LOCK("잠금 대상 문자열", time out 시간)`으로 Named Lock을 설정하고, RELEASE_LOCK으로 잠금을 해제할 수 있다.
